bin/kafka-run-class.sh

```diff
@@ -62,80 +62,80 @@ do
   if [ -z "$CLASSPATH" ] ; then
     CLASSPATH="$dir/*"
   else
-    CLASSPATH="$CLASSPATH:$dir/*"
+    CLASSPATH="$dir/*":$CLASSPATH
```

Comment:
It matters because if you want to override specific jars (e.g. with newer ones that are binary compatible but have bug fixes, such as compression libraries), you'd use classpath ordering to guarantee you load classes from the newer version rather than the older one.